### PR TITLE
Jellis/seek io

### PR DESCRIPTION
**Pull request by ellisjoe**

### CODE REVIEW

**Comment by ellisjoe** [comment]
@markelliot you prefer `getPosition`?

File: seek-io/src/main/java/com/palantir/seekio/SeekableInput.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Seeks to the given offset in the stream.
+     */
+    void seek(long offset) throws IOException;
+
+    /**
+     * Gets the current byte offset in the stream.
+     */
+    long getPos() throws IOException;
```
Comment (markelliot) [inline seek-io/src/main/java/com/palantir/seekio/SeekableInput.java:37]: naming to be compatible with Hadoop?

File: seek-io/src/main/java/com/palantir/seekio/SeekableInput.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * @see java.io.InputStream#read(byte[], int, int)
+     */
+    int read(byte[] bytes, int offset, int length) throws IOException;
+
+    /**
+     * @see java.io.InputStream#read()
+     */
+    int read() throws IOException;
```
Comment (markelliot) [inline seek-io/src/main/java/com/palantir/seekio/SeekableInput.java]: if not required prefer to omit

**Review by markelliot — APPROVED** [review]

File: seek-io/src/main/java/com/palantir/seekio/SeekableInput.java (hunk 1 above)
Comment (ellisjoe) [reply seek-io/src/main/java/com/palantir/seekio/SeekableInput.java:37]: yes: https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/Seekable.java#L41

File: seek-io/src/main/java/com/palantir/seekio/SeekableInput.java (hunk 2 above)
Comment (ellisjoe) [reply seek-io/src/main/java/com/palantir/seekio/SeekableInput.java]: 👍

**Review by markelliot — APPROVED** [review]

**Review by markelliot — APPROVED** [review]

**Comment by markelliot** [comment]
I'm not an approved reviewer, apparently.

**Comment by ellisjoe** [comment]
added you as an admin

**Pull request merged**